pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -156,7 +156,15 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 
 	// provide any config overrides
 	cfg.NodeName = options.NodeName
-	cfg.StreamingConnectionIdleTimeout = 5 * time.Minute // TODO: should be set
+	streamingConnectionIdletimeout, ok := options.KubeletArguments["streaming-connection-idle-timeout"]
+	if ok && len(options.KubeletArguments["streaming-connection-idle-timeout"]) > 1 {
```

Comment:
I see one issue here that cmdflags.Resolve(options.KubeletArguments, server.AddFlags) is resetting the time out to 0 if the block "kubeletArguments:" is not specified in the config file.